src/test/java/com/iexec/worker/tee/pre/MainTests.java

```diff
@@ -0,0 +1,43 @@
+package com.iexec.worker.tee.pre;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.org.webcompere.systemstubs.SystemStubs.catchSystemExit;
+
+class MainTests {
+
+    @BeforeEach
+    void init() {
+        Main.preComputeAppRunner = null;
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9})
```

Comment:
Why do you check values out of specification ?
`PreComputeAppRunner#start` should never return values not in `[0, 1, 2, 3]` in the current implementation.

Reply:
"Just in case"
But you're right, that's not needed. Let's remove these values. Thanks :)